Cargo.toml

```diff
@@ -21,6 +21,7 @@ futures = "0.3.18"
 zip = "0.5"
 thiserror = "1.0"
 serde_derive = "1.0"
+regex = "1.5.4"
```

Comment:
I think for now we can remove this dependency. Remove the `ignore_files` from `fpm::process_dir`.

Reply:
We need it for getting the latest version of the file in `.history` folder and comparing it with the current one